bots/example.py

```diff
@@ -0,0 +1,47 @@
+import os
+from datetime import timedelta
+
+from ape_ethereum import multicall
+from silverback import SilverbackApp
+
+from apepay import StreamManager
+
+app = SilverbackApp()
+# NOTE: You should use one bot per-supported network
+# NOTE: This bot assumes you use a new bot per deployment
+sm = StreamManager(os.environ["APEPAY_CONTRACT_ADDRESS"])
+
+# NOTE: You would probably want to index this by network and deployment,
+#       if you were operating on multiple networks or deployments
```

Comment:
question: You can do that with Silverback?

Reply:
This note is a shoutout to the database, that you would probably want to add a column for your network and deployment address for your list of streams